docs/KVSet/KVSet.v1.0.0.schema.json

```diff
@@ -0,0 +1,105 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://azconfig.io/schemas/KVSet/v1.0.0/KVSet.json",
+  "title": "An Azure App Configuration schema for importing/exporting using 'appconfig/kvset' profile",
+  "type": "object",
+  "required": [
+    "items"
+  ],
+  "properties": {
+    "items": {
+      "$id": "#/properties/items",
+      "type": "array",
+      "title": "Items",
+      "description": "An array of key-values in the Azure App Configuration store.",
+      "items": {
+        "$ref" : "#/$defs/kv"
+      }
+    }
+  },
+  "$defs": {
+    "kv": {
+      "type": "object",
+      "title": "Key-value",
+      "description": "An individual key-value and associated metadata in the 'appconfig/kvset' profile.",
+      "properties": {
+        "key": {
+          "$id": "#/properties/key",
+          "type": "string",
+          "title": "Key",
+          "description": "The 'key' in the key-value.",
+          "examples": [
+            "background_color"
+          ],
+          "pattern": "^(.*)$",
+          "minLength": 1
+        },
+        "value": {
+          "$id": "#/properties/value",
+          "title": "Value",
+          "description": "The 'value' in the key-value.",
+          "examples": [
+            "blue"
+          ],
+          "default": null,
```

Comment:
Should we default this to ""?  At least, that's the current behavior of CLI.

```
>az appconfig kv set -n configstoredemo --key abcd
Are you sure you want to set the key:
{
  "content_type": "",
  "key": "abcd",
  "label": null,
  "tags": null,
  "value": ""
}
 (y/n): y
{
  "contentType": "",
  "etag": "Zh6Xg2J73h0qxoSYuRW0Q7J8CsL",
  "key": "abcd",
  "label": null,
  "lastModified": "2021-12-03T20:30:00+00:00",
  "locked": false,
  "tags": {},
  "value": ""
}
```

Reply:
I think it is the server that assigns `""` when value is not specified.
Regardless, I think we should have it as a default for this schema too.